### ISSUE TITLE
Bug marking correct answers as wrong

### ISSUE THREAD

**Issue by abatey-itv** [zjosua/anki-mc#15]
Viewing the questions is fine. 

When I select an answer for a single choice question or multiple answers for a multiple choice questions, then I show the answer, it's always wrongly marketing my selections as wrong. 

I think that problem might be than when I hit 'show answer' it deselects my answers in the questions. 

<img width="399" alt="Screen Shot 2020-09-11 at 15 36 15" src="https://user-images.githubusercontent.com/18442874/92938518-8c7a4c00-f444-11ea-8325-d7aeac9447f0.png">

Eg in this example I selected the correct answer before pressing 'Show answer' but as you can see it has been deselected and I got the answer wrong.

Please let me know how to resolve this problem.

**Comment by zjosua** [comment]
> I think that problem might be than when I hit 'show answer' it deselects my answers in the questions.

Yes that's probably it. I can't reproduce this though.
Are you using a mobile device or is this in the desktop app?

**Comment by Heinz2411** [comment]
The problem exists with iOS mobile and tablet and not with the desktop version.
The reason probably is that when selecting the answer Anki immediately switches to the answer page of the card.
That does not leave enough time to store the selected answer to persistent data. 
As a result the selection is not available and therefore can't be evaluated correctly.
The difference to the desktop version is the missing button 'Show answer' which allows to first make the choice and afterwards switch to the answer page.
In other words: the problem is caused by the feature of not having that dedicated 'Show answer' button.
Fortunately, persistent data looks good as the scrambled order is passed correctly from question to answer page.
Providing a 'Show answer' button or limited region of the screen should fix that problem.
I couldn't find any option that would allow that.
(Anki for Mac 2.1.35, Anki for iPhone 2.0.67, Anki for iPad 2.0.67)

**Comment by Schlauer-Hax** [comment]
I found a workaround: Hold the text next to it to select it and then you can press the boxes and stuff. You maybe have to disable showing the next answer in the settings.

**Comment by zjosua** [comment]
Duplicate of #11
